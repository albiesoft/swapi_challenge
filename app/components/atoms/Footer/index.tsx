import { FC } from 'react';
import { Layout } from 'antd';

const Footer: FC = () => {
  const { Footer } = Layout;
  return (
    <Footer>
      Made with <img src="/heart.svg" alt="love" width="30px" /> by Albert
      Fernandez
      <style jsx>{`
        text-align: center;
        img {
          padding-left: 5px;
        }
      `}</style>
    </Footer>
  );
};

export default Footer;
