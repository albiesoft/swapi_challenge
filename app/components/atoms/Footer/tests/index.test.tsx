import { shallow, ShallowWrapper } from 'enzyme';
import Footer from '..';

describe('<Footer />', () => {
  test('It renders correctly', () => {
    const target: ShallowWrapper = shallow(<Footer />);
    expect(target).toMatchSnapshot();
  });
});
