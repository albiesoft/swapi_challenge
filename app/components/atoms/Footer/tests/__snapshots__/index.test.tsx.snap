// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> It renders correctly 1`] = `
<Footer>
  Made with 
  <img
    alt="love"
    src="/heart.svg"
    width="30px"
  />
   by Albert Fernandez
  <style>
    
        text-align: center
        img {
          padding-left: 5px;
        }
      
  </style>
</Footer>
`;
